v2/sender.go

```diff
@@ -103,6 +104,15 @@ func SetMessageTTL(ttl time.Duration) func(msg *azservicebus.Message) error {
 	}
 }
 
+// SetMessageTrace inject the spanContext inside the input context into a service bus message
+func SetMessageTrace(ctx context.Context) func(msg *azservicebus.Message) error {
```

Comment:
could you move this func to the tracing package?

so to explicitely propagate the trace on message send, it will look like

`sender.Send(ctx, msg, tracing.PropagateFrom(ctx))`